source/stdcpp/test/vector.d

```diff
@@ -5,13 +5,38 @@
 *******************************************************************************/
 
 module stdcpp.test.vector;
-
 import stdcpp.vector;
+import stdcpp.allocator;
+int a = 42;
+int b = 9;
+allocator!int alloc_instance = allocator!(int).init;
```

Comment:
We need to remove this. I think a D constructor that forwards to the C++ constructor makes sense.